addons/hr_holidays/static/src/img/icons/Annual_Time_Off.svg

```diff
@@ -0,0 +1,21 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
```

Comment:
Why the images are duplicated (png + svg) ? 

The svg files seem to be unused.